.stickler.yml

```diff
@@ -0,0 +1,17 @@
+linters:
+    eslint:
+        config: './eslint.config'
+fixers:
+    enable: true
+    
+files:
+  ignore:
+    - 'bower_components/*'
+    - 'node_modules/*'
+    - '.eslintrc.json'
+    - 'webpack.config.js'
+    - 'dist/*'
+    - 'testing/*'
+
+
+
```

Comment:
you should not be ignoring `./estilnrc.json` as that is the one that should be listened to in line 3. Also this si fundamentally different from [This](https://github.com/microverseinc/linters-config/blob/master/javascript/.stickler.yml). Please refer to the comment on the eslint.config file.